src/index.js

```diff
@@ -1,2 +1,57 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+const initialCount = 0;
+
+function render(count) {
+  function handleClick() {
+    render(count + 1);
+  }
+
+  function handleClickNumber(value) {
+    if (Number.isNaN(Number(value))) {
+      const { assert } = console;
+      return assert(false, { value, message: '숫자가 아닙니다.' });
```

Comment:
return 하는 것을 보면 `assert`의 결과를 반환하는 것처럼 보여요. 명확하게 작성해야 합니다.

```suggestion
      assert(false, { value, message: '숫자가 아닙니다.' });
      return
```

Reply:
아... 반환할 의도가 아니니까 return을 저렇게 쓰면 이상하군요.. 감사합니다 :)